otherlibs/action-plugin/test/depends-on-directory-with-glob/run.t

```diff
@@ -1,3 +1,5 @@
+  $ export DUNE_CACHE=disabled
+
```

Comment:
Why's this necessary (here and elsewhere)? Unsetting `DUNE_CACHE` seems enough here: `time CI=true ./dune.exe runtest otherlibs/action-plugin/test/depends-on-directory-with-glob`...